client/src/app/pages/search/components/SearchTabs.tsx

```diff
@@ -22,9 +22,13 @@ import HelpIcon from "@patternfly/react-icons/dist/esm/icons/help-icon";
 
 export interface SearchTabsProps {
   filterPanelProps: {
```

Comment:
```suggestion
    filterPanelProps: {
    advisoryFilterPanelProps?: Record<string, unknown>;
    packageFilterPanelProps?: Record<string, unknown>;
    sbomFilterPanelProps?: Record<string, unknown>;
    vulnerabilityFilterPanelProps?: Record<string, unknown>;
  };
```
Can we try that ?

Reply:
`Record<string, unknown>` cascaded type errors. So I replaced it by 
```ts
filterPanelProps: {
    advisoryFilterPanelProps: IFilterPanelProps<
      AdvisorySummary,
      "" | "modified" | "average_severity"
    >;
    packageFilterPanelProps: IFilterPanelProps<
      PackageTableData,
      "" | "type" | "arch"
    >;
    sbomFilterPanelProps: IFilterPanelProps<SbomSummary, "" | "published">;
    vulnerabilityFilterPanelProps: IFilterPanelProps<
      VulnerabilitySummary,
      "" | "average_severity" | "published"
    >;
  };
```

This way the types are more accurate